examples/visualization/src/app/react/lazy/page.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
+
+import { Suspense, lazy } from '@suspensive/react'
+import React, { useState } from 'react'
+
+const Comp = lazy(() => import('./Comp'), { reload: 3 })
```

Comment:
```suggestion
const Comp = lazy(() => import('./Comp'), { onError: () => window.location.reload() })
```

I was worried because I was not sure if the option name reload would be easy to understand, but what if I handle it like this in onError? I think it could be used in various ways. I think I could even induce a refresh by displaying a toast.

Reply:
`@suspensive/react` should also support `react-native`, so this one, which doesn't use sessionStorage internally, seems like a good general-purpose option.

Reply:
cc. @gwansikk @kangju2000

Reply:
But if we remove `reload` and use `onError`, how can we implement retrying `window.location.reload()`? (Wouldn't it get a bit complicated?)

I agree with creating an escape hatch using `onError`, but if implementing `retry` is complicated, I think a `retry` option is also necessary.

Reply:
I think we can convert like below

### Infinite reload

```tsx
import { lazy } from '@suspensive/react'

const customLazy = lazy.create({ reload: true })
const Comp = lazy(() => import('...'), { reload: true })

import { lazy } from '@suspensive/react'

const customLazy = lazy.create({ onError: () => window.location.reload() })
const Comp = lazy(() => import('...'), { onError: () => window.location.reload() })
```

### Reload only at specific times.

Below is pseudo code.

```tsx
const MAX_RELOAD = 2

import { lazy } from '@suspensive/react'

const customLazy = lazy.create({ reload: MAX_RELOAD })
const Comp = lazy(() => import('...'), { reload: MAX_RELOAD })

import { lazy } from '@suspensive/react'

const customLazy = lazy.create({ 
  onSuccess:  ({ load }) => {
    const storageKey = load.toString()
    window.sessionStorage.removeItem(storageKey)
  },
  onError: ({ load }) => {
    const storageKey = load.toString()
    const sessionStorageReloadedCount = parseInt(window.sessionStorage.getItem(storageKey), 10)
    const reloadedCount = Number.isNaN(sessionStorageReloadedCount) ? 0 : sessionStorageReloadedCount    
    window.sessionStorage.setItem(storageKey, String(reloadedCount + 1))
    if(reloadedCount <= MAX_RELOAD) window.location.reload()
  }
})
const Comp = lazy(() => import('...'),{ 
  onSuccess:  ({ load }) => {
    const storageKey = load.toString()
    window.sessionStorage.removeItem(storageKey)
  },
  onError: ({ load }) => {
    const storageKey = load.toString()
    const sessionStorageReloadedCount = parseInt(window.sessionStorage.getItem(storageKey), 10)
    const reloadedCount = Number.isNaN(sessionStorageReloadedCount) ? 0 : sessionStorageReloadedCount    
    window.sessionStorage.setItem(storageKey, String(reloadedCount + 1))
    if(reloadedCount <= MAX_RELOAD) window.location.reload()
  }
})
```

Reply:
I made Pull Request to replace `reload` with `onSuccess` and `onError` https://github.com/toss/suspensive/pull/1658